scipy-1.0/paper.tex

```diff
@@ -1120,14 +1120,14 @@ \subsection*{Community beyond the SciPy library}
 of SciPy features, and then returning to the base SciPy library
 to suggest and even implement improvements to enable
 more effective science applications. For example, there
-are common contributors to the SciPy and astropy core
+are common contributors to the SciPy and Astropy core
 libraries\cite{astropy-2018}, and what works well for 
-one of the code bases, infrastructures, or communities 
-is often transferred in some form to the other. At the code
-base level, the \texttt{binned\_statistic} functionality
+one of the codebases, infrastructures, or communities 
+is often transferred in some form to the other. At the codebase 
+level, the \texttt{binned\_statistic} functionality
```

Comment:
"code base" -> "codebase" only for consistency throughout paper (both are valid)